docs/contents/proposals/secrets/secrets.md

```diff
@@ -37,24 +115,47 @@ secretRefs:
     - action: CreateDBCluster
 ```
 
-These API specific YAML files will be available for users to download and save into their directories of choice.
-Let us assume the user Alice wants to use the Amazon RDS API to create a new DB Instance. Alice must create a Kubernetes Secret in the desired namespace and save the RDS YAML file in the directory she would like the DBInstance to reside in. Then, she simply calls the kubectl apply command, passes in her specification, and waits for a response from the Kubernetes API server.
+### How will the user interact with the proposed solution?
+Let us assume the user Alice wants to use the Amazon RDS API to create a new DB Instance. Alice must first create a Kubernetes Secret object and specify a key within the Secret that will store her RDS DB Instance master user password. Alice will reference that Secret and key within her RDS DB Instance custom resource manifest. Alice then simply calls the [kubectl apply command](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#kubectl-apply), passes in her specification, and waits for a response from the [Kubernetes API](https://kubernetes.io/docs/concepts/overview/kubernetes-api/) server. 
 The user's process is shown below.
 
+#### User Flowchart
 ![User Flowchart](images/rds-db-instance-creation-user-flowchart.png)
 
+### How will the Kubernetes API server and ACK RDS controller replace specified fields?
 Upon Alice calling the command, the Kubernetes API server writes the new CR to storage. 
-The ACK RDS controller detects the change, reads the CR, and calls the Kubernetes API to retrieve the Secret information. The controller then reads the output, decodes the Secret value, and calls the createDBInstance method from the RDS API, passing in the decoded Secret for the field MasterUserPassword. The controller will store the DB instance identifier with the Secret reference and field name for future use, and write the CR for the new DB instance. Lastly, the Kubernetes API server receives an updated status and sends back a response to the user.
+The ACK RDS controller detects the change, reads the CR, and calls the Kubernetes API to retrieve the Secret information. The controller then reads the output, decodes the Secret value, and calls the createDBInstance method from the RDS API, passing in the decoded Secret for the field MasterUserPassword. Lastly, the Kubernetes API server receives an updated status.
 These processes are shown below.
 
-![Kubernetes API and ACK RDS Controller Flowcharts](images/rds-db-instance-creation-sequence-backend-flowchart.png)
+#### Kubernetes API and ACK RDS Controller Flowcharts
```

Comment:
Feel free to write out the steps for these flowcharts in a numbered list. This will both help visually impaired readers and be a way for you to internalize and validate the steps themselves.